add title to carousel images

vincerubinetti opened the pull request:
![image](https://user-images.githubusercontent.com/8326331/65072254-84d1b580-d95e-11e9-9a35-fa6d7e74670e.png)

- adds paper title and link to manubot html version of paper to catalog carousel images on hover
- tweaks the carousel behavior code to accommodate the above
- removes globe icon from catalog page thumbnails
- tweaks and cleans some css

dhimmel commented:
IT looks like a cache file has been tracked.. should a pattern to prevent that go in .gitignore?

vincerubinetti commented:
> IT looks like a cache file has been tracked.. should a pattern to prevent that go in .gitignore?

I updated the packages in the repo to the latest versions, and I think maybe they changed the name from `.jekyll-metadata` to `.jekyll-cache`, but the gitignore didn't update. I updated it now.

vincerubinetti commented:
@dhimmel ready for review and merge
